src/app/utilities/transmart-utilities/transmart-cohort-mapper.ts

```diff
@@ -0,0 +1,55 @@
+import {Cohort} from '../../models/cohort-models/cohort';
+import {TransmartConstraintMapper} from './transmart-constraint-mapper';
+import {TransmartCohort} from '../../models/transmart-models/transmart-cohort';
+
+export class TransmartCohortMapper {
```

Comment:
this should be also merged with [GbBackendMapper](src/app/utilities/gb-backend-mapper.ts), right?

Reply:
indeed